posts/five-interns-walk-into-an-office.md

```diff
@@ -0,0 +1,44 @@
+#### Yashar
+Great culture, atmosphere, and people; Ippon has it all. The continuous-learning culture that Ippon fosters encouraged me to dive head first into several popular technologies. From web technologies such as Angular and  Spring coupled through JHipster to increasingly relevant cloud powerhouse like Amazon Web Services, stimulating projects and talented supervisors made the learning process challenging but rewarding. During the 3 months I interned at Ippon, I learned and experienced more about agile development and how the development process works in a business setting than I have in 2 years as a computer science student. Scrum method combined with a fast-paced application of knowledge made possible by extremely intelligent and willing co-workers introduced me to a whole new domain of software development. These same co-workers make the work environment and culture at Ippon unlike anything I have been a part of at any job. We work hard together but we also have fun together. Casual Fridays where we socialize over drinks and disco balls as well as fun social events were things I would have never expected to get from a technical internship. I am very grateful for the experiences I had at Ippon and look forward to building upon them while continuing my career.
+#### Auguste
+As a French engineering student, I wanted to do a summer internship in the US to discover American culture, work around American citizens, and improve my speaking skills. In a way, I was pursuing the American way of life. Working at Ippon USA during the summer turned out to be a unique experience in several ways. First, the US headquarters is located in Richmond, not the place you dreamed of your entire life. Despite its smaller size, Virginia’s state capital has a lot to offer. I found it to be a great mix between the warm Southern hospitality and the Northern open-mindedness. Second, Ippon is not like any other consulting company. Like all mentioned before, the culture is what makes Ippon stand above its competitors. Staff enjoy coming to work, stay late to chat with coworkers, and plan activities on the weekend together. This environment is highly beneficial for communication, productivity, and company loyalty. As an inexperienced intern I did not had the opportunity to work for a client. Instead we worked on our own intern project which had a lot of advantages. It was completely free and a great guide to help us progress both as a team and as individuals. We learned a lot state-of-the-art technology and we got to work with and hang out with amazing people. I had a unique and genuine American experience working at Ippon and I would strongly recommend any student who would like to do a summer internship to apply to Ippon USA.
```

Comment:
We learned a lot state-of-the-art technology -> We learned a lot of state-of-the-art technologies